core/domain/feedback_jobs_continuous_test.py

```diff
@@ -436,12 +439,16 @@ class RealtimeFeedbackAnalyticsUnitTests(test_utils.GenericTestBase):
     ALL_CC_MANAGERS_FOR_TESTS = [MockFeedbackAnalyticsAggregator]
 
     def _get_swap_context(self):
+        """Swaps the context of jobs registry."""
         return self.swap(
             jobs_registry, 'ALL_CONTINUOUS_COMPUTATION_MANAGERS',
             self.ALL_CC_MANAGERS_FOR_TESTS)
 
     def _flush_tasks_and_check_analytics(
             self, exp_id, expected_thread_analytics_dict):
+        """Processes and flushes the pending tasks and checks the thread
```

Comment:
Looks like there are too many ands! 

```suggestion
        """Processes and flushes the pending tasks, then checks the thread
```

Reply:
Done